cddl/corim-frags.mk

```diff
@@ -76,3 +76,19 @@ CORIM_FRAGS += validity-map.cddl
 CORIM_FRAGS += $(COMID_FRAGS)
 
 CORIM_EXAMPLES := $(wildcard examples/corim-*.diag)
+
+INTREP_FRAGS := intrep-acs.cddl
```

Comment:
Just a minor nit,

The CDDL names in our repo are long but quite expressive, for example:
`measured-element-type-choice.cddl`

I would request, we rename INTREP everywhere to 

`INTERNAL-REP`

So the CDDL becomes: internal-rep-claims-map.cddl

This avoids any confusion in the naming to mis-interpret it as Integer Represenation!

Reply:
> This avoids any confusion in the naming to mis-interpret it as Integer Representation!

No way, that's a feature, not a bug :-) I made it expressly to sound like INTREPid!

Reply:
But it is not INTREPID after-all ??